### ISSUE TITLE
HTML gets parsed by linter-jshint / add option to only lint .js files

### ISSUE THREAD

**Issue by jelmerdemaat** [AtomLinter/linter-jshint#50]
**Issue**
For some reason, my HTML documents also get parsed by linter-jshint, and I don't see any option to give some file type inclusions/exclusions to jshint. See also the screenshot below.

My HTML uses [grunt-include-replace](https://github.com/alanshaw/grunt-include-replace), the syntax of which linter-jshint seems to incorrectly flag as "wrong JavaScript".

**What would solve this for me**
Don't let HTML be parsed by linter-jshit, or give a settings possibility to include/exclude file types.

**Screenshot**
![image](https://cloud.githubusercontent.com/assets/942560/6226885/1d86da2c-b69a-11e4-90aa-4fe38f9bbed5.png)

**Technical details**
Atom version 179
Windows 7
```
$ apm -v
apm  0.137.0
npm  2.3.0
node 0.10.35
git 1.9.5.msysgit.0

$ apm list
advanced-new-file@0.3.3
atom-color-highlight@3.0.8
auto-indent@0.1.0
autocomplete-paths@1.0.2
autocomplete-plus@2.1.6
caniuse@0.5.3
emmet@2.3.3
linter@0.11.1
linter-jscs@1.5.0
linter-jshint@0.1.0
linter-php@0.0.11
merge-conflicts@1.2.10
minimap@4.2.2
pretty-json@0.3.2
seti-ui@0.6.3
```

**Comment by jelmerdemaat** [comment]
Still experiencing this, any thoughts?

**Comment by tjdraper** [comment]
I'm experiencing this as well. It makes it frustrating for templating curly braces in CMS HTML files. Really need a way to disable.

**Comment by steelbrain** [comment]
Fixed
